xml/tuning_dynamic_debug.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-tuning-dynamic-debug">
+  <title>Dynamic debug&mdash;dynamically enable/disable kernel debug-print code</title>
```

Comment:
I'd suggest a different title for this chapter, as it's goal is to present an overview of the dynamic debug feature, not only how to enable/disable it.  How about something like "Dynamic debug - Kernel debugging messages"?  Or "Dynamic debug - Kernel messages"?